docs/guides/deployment/heroku.rst

```diff
@@ -0,0 +1,99 @@
+.. _ref_guide_deployment_heroku:
+
+=========
+On Heroku
+=========
+
+In this guide we show how to deploy EdgeDB to Heroku using a Heroku PostgreSQL
+add-on as the backend.
+
+Because of Heroku's architecture EdgeDB must be deployed with a web app on
+Heroku. For this guide we will use a `todo app written in node <todo-repo_>`_.
+
+.. _todo-repo: https://github.com/edgedb/simpletodo/tree/main
+
+First copy the code, initialize a new git repo, and create a new heroku app.
```

Comment:
They'll have to install Node + npm to have access to `npx` down the line. May want to note this and link to https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-version-manager-to-install-node-js-and-npm

Reply:
changed